site/content/contribute/webapp/end-to-end-tests.md

```diff
@@ -43,6 +43,8 @@ The folder structure is mostly based on the [Cypress scaffold](https://docs.cypr
     - To start writing tests,
         - simply create a new file (e.g. `login_spec.js`) within `/e2e/cypress/integration` folder and;
         - refresh tests list in the Cypress Test Runner and a new file should have appeared in the list.
+    - Subfolder naming convention depends on test grouping, which is usually based on the name of the "Sheet Tab" in the Release testing spreadsheet (e.g. `/e2e/cypress/integration/messaging/` for "Messaging" sheet tab).
```

Comment:
Just a couple small suggestions: Maybe instead of `"Sheet Tab"` just `tab` (no quotes, lowercase, no "sheet")? And cap "Testing", so with the changes it would read:

... `usually based on the name of the tab in the Release Testing spreadsheet (e.g. `/e2e/cypress/integration/messaging/` for "Messaging" tab).`

But it looks good; I'll approve it and let you make the final decision on my suggestions. :)

Reply:
Thanks @lindalumitchell, updated per recommendation.